packages/validate/src/ValidateMixin.js

```diff
@@ -211,30 +214,79 @@ export const ValidateMixin = dedupeMixin(
             'submitted',
             'prefilled',
             'label',
-          ],
-          _onErrorShowChangedAsync: ['errorShow'],
-        };
+          ].some(key => changedProperties.has(key))
+        ) {
+          this._createMessageAndRenderFeedback();
+        }
+
+        if (changedProperties.has('errorShow')) {
+          this._onErrorShowChangedAsync();
+        }
       }
 
-      static get syncObservers() {
-        return {
-          ...super.syncObservers,
-          validate: [
+      _requestUpdate(name, oldVal) {
+        super._requestUpdate(name, oldVal);
+
+        /**
+         * Validation needs to happen before other updates
+         * E.g. formatting should not happen before we know the updated errorState
+         */
+        if (
+          [
             'errorValidators',
             'warningValidators',
             'infoValidators',
             'successValidators',
             'modelValue',
-          ],
-          _onErrorChanged: ['error'],
-          _onWarningChanged: ['warning'],
-          _onInfoChanged: ['info'],
-          _onSuccessChanged: ['success'],
-          _onErrorStateChanged: ['errorState'],
-          _onWarningStateChanged: ['warningState'],
-          _onInfoStateChanged: ['infoState'],
-          _onSuccessStateChanged: ['successState'],
-        };
+          ].some(key => name === key)
+        ) {
+          this.validate();
+        }
+
+        // @deprecated adding css classes for backwards compatibility
+        this.constructor.validationTypes.forEach(type => {
```

Comment:
move the classes to async

Reply:
tried to move but fails a test in a strange way 😭 
gonna leave if for now like this... we can't save the whole world in one PR 😬